ci-operator/step-registry/single-node/recert/single-node-recert-commands.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+recert_script=$(cat << 'EOF'
+#!/usr/bin/env bash
+
+set -euoE pipefail
+
+export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig
+function wait_for_api {
+  echo "Waiting for API..."
+  until oc get clusterversion &> /dev/null
+  do
+    echo "Waiting for API..."
+    sleep 5
+  done
+  echo "API is available"
+}
+
+function start_containers {
+  systemctl start crio
+  systemctl start kubelet
+}
+
+function stop_containers {
+  systemctl stop kubelet
+  crictl ps -q | xargs crictl stop || true
+  systemctl stop crio
+}
+
+function recert {
+  RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.13.6-x86_64
+  ETCD_IMAGE="$(oc adm release extract --from="$RELEASE_IMAGE" --file=image-references | jq '.spec.tags[] | select(.name == "etcd").from.name' -r)"
+  RECERT_IMAGE="quay.io/otuchfel/recert:latest"
```

Comment:
```suggestion
  RECERT_IMAGE="quay.io/recert/recert:latest"
```

Reply:
I was thinking that we would like to test the `pipeline:recert` image, which should be the image just built from the current PR. I am currently testing exactly how we can achieve that and I'll push new changes.

In addition, I have added the [promotion](https://github.com/openshift/release/pull/43149/files#diff-c4eb3c4428cadd4e67617294957d57a6c9bd5df77c9a62181e46c87d0c845c64R45-R48) of the image to be under `quay.io/edge-infrastructure/recert` (as I think most, if not all, projects under `rh-ecosystem-edge` push their images under `edge-infrastructure`). WDYT?

Reply:
sgtm

Reply:
I don't think the promotion is sufficient to get you  to quay, you also need to add a mapping to `release/core-services/image-mirroring/edge-infrastructure`

Reply:
Let's have RECERT_IMAGE default to `quay.io/recert/recert:latest` in the absence of a prow configuration, so the script can still be used outside of prow

Reply:
> Let's have RECERT_IMAGE default to quay.io/edge-infrastructure/recert:latest in the absence of a prow configuration, so the script can still be used outside of prow

Exactly, this is what I'm currently testing :+1: